src/libmongoc/src/mongoc/mongoc-client-private.h

```diff
@@ -42,8 +42,8 @@
 BSON_BEGIN_DECLS
 
 /* protocol versions this driver can speak */
-#define WIRE_VERSION_MIN 3
-#define WIRE_VERSION_MAX 13
+#define WIRE_VERSION_MIN 3	/* a.k.a. minWireVersion */
+#define WIRE_VERSION_MAX 14	/* a.k.a. maxWireVersion */
```

Comment:
Please run clang-format. I think the tab will be replaced with spaces.

Reply:
Sorry, I meant to make this a draft PR-- I am wondering if there are tests that need changing. I only see a couple that reference maxWireVersion 13, I'll see if it causes a regression.